nucliadb_core/src/query_planner.rs

```diff
@@ -175,10 +176,17 @@ impl From<SearchRequest> for QueryPlan {
 
 fn compute_pre_filters(search_request: &SearchRequest) -> Option<PreFilterRequest> {
     let mut pre_filter_request = PreFilterRequest {
+        security: None,
         timestamp_filters: vec![],
         labels_filters: vec![],
     };
 
+    // Security filters
+    let request_has_security_filters = search_request.security.is_some();
```

Comment:
make sure this is correct! should we check with something similar to python's `.HasField("security")`?